mlrun/utils/helpers.py

```diff
@@ -1015,16 +1015,32 @@ def create_class(pkg_class: str):
     return class_
 
 
-def create_function(pkg_func: str):
+def create_function(pkg_func: str, need_to_reload: bool = False):
```

Comment:
```suggestion
def create_function(pkg_func: str, reload: bool = False):
```

Reply:
`reload` is already exists when `from importlib import import_module, reload`

Reply:
`reload_modules` then?
I just don't like the "need to" part